src/endpoints/Coverage/types.ts

```diff
@@ -4,16 +4,17 @@ import {
     CultureRef,
     CountryRef,
     Contact,
-    Coverage,
+    CoverageEntry,
```

Comment:
Had to rename this to fix naming collision: `Coverage` endpoint ES namespace vs `Coverage` entity